_episodes/05-mplhep.md

```diff
@@ -269,6 +269,12 @@ fig.show()
 {: .challenge}
 
 > ## Solution
+> Theory:
+> First, rest mass is not a conserved quantity.
+> The important idea is that in Higgs rest frame two Z bosons of 90 GeV each cannot be produced.
+> Secondly, the Higgs decays to one Z boson which is on-shell whereas the other Z boson is off-shell.
```

Comment:
Recently the Time-energy uncertainty argument (in context of particles and energy conservation) has drawn some criticism recently, for instance in Griffiths and others.

Energy of Z boson (or any decaying particle) is the expectation value of energy which is inherently probabilistic. Energy and momentum are always conserved. 

So I think it would be good not to use that argument.

Reply:
Thanks, that's good to know about the time-energy uncertainty!